Sprint-3/todo-list/script.js

```diff
@@ -1,14 +1,41 @@
+let todos = [];
+
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
-}
 
-// These are the same todos that currently display in the HTML
-// You will want to remove the ones in the current HTML after you have created them using JavaScript
-let todos = [
-  { task: "Wash the dishes", completed: false },
-  { task: "Do the shopping", completed: false },
-];
+  //clear old items
+  list.innerHTML = "";
+
+  //loop through array and display tasks to user
+  todos.forEach((todoObject) => {
+    //create new li
+    const li = document.createElement("li");
+
+    //create checkox toggle
+    const checkbox = document.createElement("input");
+    checkbox.type = "checkbox";
+    checkbox.checked = todoObject.completed;
+
+    // clicked, toggle competed or not completed
+    checkbox.addEventListener("change", () => {
```

Comment:
Using a checkbox like this works pretty well here - and checking the box does have the effect of marking the todo as done!

The instructions do say to use

```html
  <i class="fa fa-check" aria-hidden="true"></i>
  <i class="fa fa-trash" aria-hidden="true"></i>
```

here to show a checkbox and a trash symbol for deleting a todo.